### PR TITLE
refactor: vehicle type without networking

### PR DESCRIPTION
**Pull request by MoskalykA**
Hello, the `GetVehicleClassFromName` function gives a reference list so I suggest not doing any more networking and just finding the right value on the server side.

### CODE REVIEW

**Comment by Arctos2win** [comment]
> Hello, the `GetVehicleClassFromName` function gives a reference list so I suggest not doing any more networking and just finding the right value on the server side. 

what about addon vehicles

File: [core]/es_extended/server/functions.lua [hunk 1]
```diff
@@ -287,21 +287,471 @@ function ESX.GetIdentifier(playerId)
...
-
-	if Core.vehicleTypesByModel[model] then
-		return cb(Core.vehicleTypesByModel[model])
-	end
-
-	ESX.TriggerClientCallback(player, "esx:GetVehicleType", function(vehicleType)
-		Core.vehicleTypesByModel[model] = vehicleType
-		cb(vehicleType)
-	end, model)
+local classFromName = {
```
Comment (Rav3n95) [inline [core]/es_extended/server/functions.lua:290]: Are you mental?

Comment (iSentrie) [reply [core]/es_extended/server/functions.lua:290]: jeez

**Comment by MoskalykA** [comment]
> what about addon vehicles

I hadn't thought of that, but I could turn this pull request into a draft and make a pull request on fivem to implement server-side detection.
However, I find it unacceptable to be insulted on the pretext of a small error, especially coming from the leader of your team.

**Comment by TheFantomas** [comment]
@MoskalykA Can close that? 

It is fixed in the latest version. It is not the best solution, but unfortunately there is no other solution for now, I would say.

**Pull request closed without merging**